modules/51DegreesRtdProvider.md

```diff
@@ -0,0 +1,130 @@
+# 51Degrees RTD Submodule
+
+## Overview
+
+    Module Name: 51Degrees Rtd Provider
+    Module Type: Rtd Provider
+    Maintainer: support@51degrees.com
+
+## Description
+
+51Degrees module enriches an OpenRTB request with [51Degrees Device Data](https://51degrees.com/documentation/index.html).
+
+51Degrees module sets the following fields of the device object: `make`, `model`, `os`, `osv`, `h`, `w`, `ppi`, `pixelratio` - interested bidder adapters may use these fields as needed. In addition the module sets `device.ext.fiftyonedegrees_deviceId` to a permanent device ID which can be rapidly looked up in on premise data exposing over 250 properties including the device age, chip set, codec support, and price, operating system and app/browser versions, age, and embedded features.
+
+The module supports on premise and cloud device detection services with free options for both. 
+
+A free resource key for use with 51Degrees cloud service can be obtained from [51Degrees cloud configuration](https://configure.51degrees.com/tWrhNfY6).  This is the simplest approach to trial the module.
+
+An interface compatible self hosted service can be used with .NET, Java, Node, PHP, and Python.  See [51Degrees examples](https://51degrees.com/documentation/_examples__device_detection__getting_started__web__on_premise.html).
+
+Free cloud and on premise solutions can be expanded to support unlimited requests, additional properties, and automatic daily on premise data updates via a [subscription](https://51degrees.com/pricing).
+
+## Usage
+
+### Integration
+
+Compile the 51Degrees RTD Module with other modules and adapters into your Prebid.js build:
+
+```
+gulp build --modules="rtdModule,51DegreesRtdProvider,appnexusBidAdapter,..."  
+```
+
+> Note that the 51Degrees RTD module is dependent on the global real-time data module, `rtdModule`.
+
+### Prerequisites
+
+#### Resource Key
+In order to use the module please first obtain a Resource Key using the [Configurator tool](https://configure.51degrees.com/tWrhNfY6) - choose the following properties:
```

Comment:
Configurator is a rather awkward word, I recommend you choose a word more common in English